common/src/main/java/io/netty/util/internal/MpscLinkedQueueNode.java

```diff
@@ -18,6 +18,10 @@
 
 import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
 
+/**
+ * @deprecated will be removed as not used anymore inside netty.
```

Comment:
This class is internal. My vote is to remove it if we no longer need it. Lean and mean code base.